src/utils/sqs_consumer.ts

```diff
@@ -75,10 +75,12 @@ export class SqsConsumer {
         }
 
         // Handle message
+        let error: Error | undefined;
         try {
             await this._handleMessage(message);
         } catch (err) {
-            logger.error({ err, message, id: this._id }, 'Encountered error while handling message');
+            error = err;
```

Comment:
why is the reassignment necessary?

Reply:
Just making the `err` available outside of the `catch` scope